inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_resample_test.hpp

```diff
@@ -126,6 +154,8 @@ TEST_P(myriadResampleLayerTests_smoke, Resample) {
 
     std::map<std::string, std::string> params;
     params["antialias"] = std::to_string((int)antialias);
+    params["coordTransMode"] = std::to_string(coordTransMode);
```

Comment:
What values this parameter can have? Integer or text like "half_pixel"?

Reply:
Here, in case of tests, Integer.

Reply:
but what does parser of layer get? Does it take text representation of parameters?

Reply:
Parser gets text representation, but in tests it is easier to use it's integer value (according to the enum declaration), IMO

Reply:
when you fulfill map `params`, "cordTransMoe" = "some_int_value". Parser gets such line, not text representation. The test create parameters for the layer, and these parameters are integers converted to string. Or I am mistaken?

Reply:
Yes, you are correct.
Found where I was mistaken. After fulfilling map params, I do not process these values in parseResample method, but assign these parameters the default value, that is why it acts normal and why i didn't really looked at the parameters map before -> gets text and works fine. Interpolate stage calls addResampleNearestStage with correct parameters, but Resample - with the default ones.